src/lib/y2storage/setup_checker.rb

```diff
@@ -55,43 +55,43 @@ def initialize(devicegraph)
     #
     # @return [Boolean]
     def valid?
-      errors.empty?
+      errors.empty? && warnings.empty?
     end
 
-    # Whether the system would be bootable using current setup
+    # Whether the system contain fatal errors preventing to continue
     #
-    # @return[Boolean] true if the system is bootable; false otherwise.
-    def bootable?
-      boot_errors.empty?
+    # @return [Array<SetupError>]
+    def errors
+      boot_requirements_checker.errors
     end
 
-    # All storage errors detected in the setup, for example, when a /boot/efi partition
-    # is missing in a UEFI system, or a valid partition for root is missing.
+    # All storage warnings detected in the setup, for example, when a /boot/efi partition
+    # is missing in a UEFI system, or missing separate /boot.
```

Comment:
NP: I would mention a case where a required product volume is missing: `in a UEFI system, or a required product volume (e.g., swap) is missing.`